### PR TITLE
Use RemoteConfiguration in UiTheme

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
Create a base mechanism to use RemoteConfiguration in UiTheme

MOB-1571

### CODE REVIEW

**Review by DavDo — APPROVED** [review]

**Review by gugalo — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/UiTheme.java [hunk 1]
```diff
@@ -1404,202 +1424,252 @@ public String getAppBarTitle() {
         return appBarTitle;
     }
 
+    @Deprecated
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/UiTheme.java:1427]: You will probably need to add docs here to explain the alternative for the deprecated function(s).

**Review by andrews-moc — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/UiTheme.java (hunk 1 above)
Comment (AndriiHorishniiMOC) [reply widgetssdk/src/main/java/com/glia/widgets/UiTheme.java:1427]: Yes and no.
We don't have documentation for those methods, so they are our internal methods.
For now, we don't have the alternatives, I just mark the methods that should be moved from here.

But I can link this method with new in the next PRs of the RemoteConfiguration after those methods will be done.

File: widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfiguration.java [hunk 2]
```diff
@@ -17,6 +18,7 @@ public class GliaSdkConfiguration {
     private final String contextAssetId;
     @Deprecated
     private final String contextUrl;
+    private final RemoteConfiguration remoteConfiguration;
```
[the remark this answers is not in the document]
Comment (AndriiHorishniiMOC) [reply widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfiguration.java:21]: Good idea!

**Comment by AndriiHorishniiMOC** [comment]
!merge

**Pull request merged**